robottelo/vm.py

```diff
@@ -56,7 +56,7 @@ class VirtualMachine(object):
 
     def __init__(
             self, cpu=1, ram=512, distro=None, provisioning_server=None,
-            image_dir=None):
+            image_dir=None, tag=None):
```

Comment:
vm tagging is a new feature implemented in this pr.  This helps tag the vm name which will help in identifying the features causing these hanging vms in ginger.